tests/test-document-fuse.sh.in

```diff
@@ -72,7 +72,12 @@ fi
 
 # Run portal manually so that we get any segfault our assert output
 # Add -v here to get debug output from fuse
-./xdg-document-portal -r &
+if [ -n "${XDP_UNINSTALLED:-}" ]; then
+    document_portal=./xdg-document-portal
+else
+    document_portal=@libexecdir@/xdg-document-portal
+fi
+"${document_portal}" -r &
```

Comment:
If the document portal is properly D-Bus-activatable (which it should be, when running installed-tests, because being D-Bus-activatable is part of the interface it is intended to provide) then running the document portal "by hand" like this is actually unnecessary, because it will be D-Bus-activated anyway. Running it in the background means that if it fails to start, the error is ignored anyway.

Perhaps this would be better:

```
if [ -n "${XDP_UNINSTALLED:-}" ]; then
    "${document_portal}" -r &
fi
# else start it via D-Bus activation as usual
```

That would have two advantages:

* if it isn't D-Bus-activatable, for example because upstream changes broke the `.service` file or a downstream forgot to package it, then the test will correctly fail
* the `@libexecdir@` substitution is no longer necessary and the shell script can go back to being directly runnable

Reply:
Does the comment "Run portal manually so that we get any segfault our assert output" not apply then?

Reply:
If I was debugging in the sort of detail that requires that, I'd expect to be using the build-time tests.

For installed-tests, I think it's reasonable to expect that debugging failures will require whole-system debugging tools, for example capturing core dumps with systemd's `systemd-coredumpd` and `coredumpctl`.